contrib/mixin/Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: all tools manifests test
+
+all: manifests
+
+tools:
+	go install github.com/google/go-jsonnet/cmd/jsonnet@latest
+	go install github.com/brancz/gojsontoyaml@latest
+
+manifests:
```

Comment:
Please specify proper target for `manifests/etcd-prometheusRules.yaml` instead of marking everything as .PHONY

Reply:
Good point, will fix this! Thanks!